model/file_info_test.go

```diff
@@ -22,179 +24,127 @@ func TestFileInfoIsValid(t *testing.T) {
 		Path:      "fake/path.png",
 	}
 
-	if err := info.IsValid(); err != nil {
-		t.Fatal(err)
-	}
+	err := info.IsValid()
+	require.Nil(t, err)
```

Comment:
why the 2-step process? whouldn't it be simpler as
```suggestion
	require.Nil(t, info.IsValid())
```
I see you are consistent in this two step process, but we are not really caring about the err value other than checking it's not nil, so I'd suggest to change it elsewhere

Reply:
Thanks, it also was bothering me, so I decided to make it like more explicit, I don't know. Someone should just say that :)